src/wp-includes/taxonomy.php

```diff
@@ -4189,6 +4189,17 @@ function _update_post_term_count( $terms, $taxonomy ) {
 			$count += (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->term_relationships, $wpdb->posts WHERE $wpdb->posts.ID = $wpdb->term_relationships.object_id AND post_status IN ('" . implode( "', '", $post_statuses ) . "') AND post_type IN ('" . implode( "', '", $object_types ) . "') AND term_taxonomy_id = %d", $term ) );
 		}
 
+		/**
+		 * Fires when a term count is updated in the database.
+		 *
+		 * @since 6.9.0
+		 *
+		 * @param int    $tt_id         Term taxonomy ID.
```

Comment:
Honestly, I'm more tempted to rename the variable in the loop to `$tt_id` for clarity but can't suggest it as it's outside the diff.

```suggestion
		 * @param int    $term          Term taxonomy ID.
```

Reply:
+1

Reply:
Did the rename to `$tt_id`, [here](https://github.com/WordPress/wordpress-develop/pull/9698/commits/9abe84f072da7bf3692797e3362dc421c563dac7).